ftw/mobilenavigation/upgrades/20150824085129_apply_prefix_to_css/cssregistry.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<object name="portal_css">
+
+  <stylesheet id="++resource++ftw.mobilenavigation/navigation.css"
+              media=""
+              enabled="True"
```

Comment:
please do not update `media`  and `enabled` - it may have been changed by a policy profile..